eng/Packages.Data.props

```diff
@@ -89,8 +89,8 @@
     <PackageReference Update="Azure.Data.SchemaRegistry" Version="1.2.0" />
     <PackageReference Update="Azure.Data.Tables" Version="12.5.0" />
     <PackageReference Update="Azure.Messaging.EventHubs" Version="5.8.1" />
-    <PackageReference Update="Azure.Messaging.EventGrid" Version="4.12.0" />
-    <PackageReference Update="Azure.Messaging.ServiceBus" Version="7.11.1" />
+    <PackageReference Update="Azure.Messaging.EventGrid" Version="4.14.1" />
+    <PackageReference Update="Azure.Messaging.ServiceBus" Version="7.13.0" />
```

Comment:
Don't forget there's one of these down by the "test/stress/perf" dependencies.   Not sure why, but we should either bump that one too or just remove them.